vagrantfiles/Vagrantfile_vars

```diff
@@ -14,6 +14,9 @@ NODE_MEMORY_SIZE_GB = ENV['NODE_MEMORY_SIZE_GB'].to_i || 1
 # currently only used by ubuntu
 $docker_version = ''
 
+# Libvirt specific config
+LIBVIRT_STORAGE_POOL = ENV['LIBVIRT_STORAGE_POOL'] || "default"
```

Comment:
Could you please add it to the env var table in the `README.md` file

Reply:
A valid request. :+1: 
Added this in https://github.com/galexrt/k8s-vagrant-multi-node/pull/59/commits/42a905ce42810d608c80309e356aa0719a787854